assets/js/hooks/cell_editor/live_editor.js

```diff
@@ -183,11 +202,13 @@ export default class LiveEditor {
   /**
    * Updates editor selection such that cursor points to the given line.
    */
-  moveCursorToLine(lineNumber) {
+  moveCursorToLine(lineNumber, column = "1") {
```

Comment:
We should expect integer here, if there's any parsing necessary we can do it on the caller side.